servers/quarkus-tests/src/main/java/org/projectnessie/quarkus/tests/profiles/BigTableTestResourceLifecycleManager.java

```diff
@@ -37,13 +36,9 @@ public void setIntegrationTestContext(DevServicesContext context) {
   @Override
   public Map<String, String> start() {
     bigTable = new BigTableBackendContainerTestFactory();
-    bigTable.startBigtable(containerNetworkId);
+    bigTable.start(containerNetworkId);
```

Comment:
Confused - how are these refactorings touching all databases part of the Maria/MySQL change?

Reply:
Makes sense, but I'd prefer to have that as a separate change.

Reply:
Okay moved all that stuff to #8553 – this PR is back to draft state.